test/spanner.ts

```diff
@@ -106,7 +240,13 @@ describe('Spanner with mock server', () => {
   const fooNotFoundErr = Object.assign(new Error('Table FOO not found'), {
     code: grpc.status.NOT_FOUND,
   });
-  const server = new grpc.Server();
+  const xGoogReqIDInterceptor = new XGoogRequestHeaderInterceptor([
+    '/google.spanner.admin',
+    '/google.spanner.admin.database.v1.DatabaseAdmin',
```

Comment:
nit: isn't this second prefix redundant due to the prefix above (`/google.spanner.admin`)?